Resources/Locale/en-US/paper/book-secret-documents.ftl

```diff
@@ -0,0 +1,38 @@
+
+    The Emergency Security Orders listed here are only executed by direct order of the Central Command, properly executed, and using all necessary codes. Otherwise the Orders must be considered invalid. Secret documents must not be seen by ANYONE but the Head of Security, and they must NEVER fall into the wrong hands.
+
+    ORDER №1: Isolate the brig from the entire station, evacuate the heads to the brig.
+    ORDER №3: Assemble all heads on the bridge under house arrest, and do not release them until a special order from the Central Command.
+    ORDER №6: Immediately arrest all members of the Science Section.
+    ORDER №7: Immediately arrest all members of the Engineering Section.
+    ORDER №8: Immediately arrest all members of the Medical Section.
+    ORDER №9: Immediately arrest all members of the Service Section.
+    ORDER №11: Immediately arrest all members of the Supply Section.
+    ORDER №13: Arrest the RD and take them to the CC.
+    ORDER №14: Arrest the CMO and take them to the CC.
+    ORDER №15: Arrest the HoP and take them to the CC.
+    ORDER №16: Arrest the CE and take them to the CC.
+    ORDER №17: Arrest the QM and take them to the CC.
+    ORDER №20: Arrest the Captain and take them to the CC.
+    ORDER №25: Carry out the arrest of all non-employees of the NT.
+    ORDER №30: Carry out immediate evacuation of the Captain from the station.
+    ORDER №31: Conduct immediate evacuation of Central Command members from the station.
+    ORDER №34: Destroy all hostile or near-hostile forces on the station.
+    ORDER №36: Carry out the execution of all prisoners in the permabrig.
+    ORDER №39: Carry out immediate isolation of all sick people.
+    ORDER №42: Conduct immediate extermination of all diseased people.
+    ORDER №44: Carry out complete isolation of departments, take away ID cards from all NT employees below the heads.
+    ORDER №45: Conduct a complete isolation of departments, take away the ID cards of all NT employees.
+    ORDER №47: Conduct a blockade of the port of arrival, arrest all new arrivals.
+    ORDER №48: Conduct a lockout of the port of departure.
+    ORDER №53: Eliminate all representatives of the Syndicate upon detection.
+    ORDER №54: Eliminate all members of the magical community upon detection.
+    ORDER №56: Eliminate all non-NT alien species on detection.
+    ORDER №59: Arrest all non-NT alien species.
+    ORDER №61: Remove the Captain immediately and transfer command of the station to the HoP.
+    ORDER №62: Remove the Captain immediately and transfer command of the station to the RD.
+    ORDER №63: Remove the Captain immediately and transfer command of the station to the CMO.
+    ORDER №64: Remove the Captain immediately and transfer command of the station to the QM.
+    ORDER №66: Remove the Captain immediately and transfer command of the station to the Clown.
+    ORDER №67: Remove the Captain immediately and transfer command of the station to the HoS.
```

Comment:
1. You're missing half the orders.
2. You gender the HOS as him.
3. "close immediately" doesn't make sense.
4. This needs vetting by admin as it contradicts rules about escalation.
5. "the Central Command" should just be Central Command.
6. Same with NanoTrasen.
7. Don't abbreviate as CC.
8. Don't begin sentences with "Otherwise".
9. A bunch of these could be misconstrued as speciesism / encourage bigotry and should be removed ("remove diseased people" / "eliminate all non-NT alien species" / "Remove all members of the magical community").
10. Commas are used incorrectly in a bunch of places in run-on sentences e.g. "conduct XYZ, do ABC" when it should be two sentences or be joined.
11. 

This also needs vetting with a game admin.

Reply:
Made some edits, the rest seems to discuss with the game admins in discord.

Reply:
Removed the orders, the document is unreadable.